pkg/report/report.go

```diff
@@ -216,6 +217,15 @@ func latestEnd(start time.Time, testcases []*Testcase) time.Time {
 // Report prints a report for TestSuites to the directory.  ftype == json | xml
 func (ts *Testsuites) Report(dir, name string, ftype Type) error {
 	ts.Close()
+
+	// Create the folder to save the report if it doesn't exist
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
I'm concerned we may be missing / ignoring certain errs.  This is only handling the error "not exist" but there are other error that are possible.  This is an area of frustration in Go.. but it is common.

See:  https://github.com/kudobuilder/kuttl/blob/main/pkg/http/client.go#L75-L82

Reply:
I see. How about returning the error? Because things like not having permission or something can not be solved from the program, though. If the file already exists we can overwrite it but I don't know if it is a good idea to do this silently.